docs/tre-admins/setup-instructions/cicd-pre-deployment-steps.md

```diff
@@ -0,0 +1,126 @@
+# Pre-deployment steps
```

Comment:
Am I right that with this document I can get the workflow setup to deploy the solution?
If yes, then we're missing `AAD_TENANT_ID` that isn't mentioned here but is required for auto bundle registration.